pkg/cli/commands.go

```diff
@@ -21,6 +21,10 @@ func AddCommands(cmd *cobra.Command, cli CLI) {
 	rookCmd.AddCommand(NewRookFlexvolumeToCSI(cli))
 	cmd.AddCommand(rookCmd)
 
+	longhornCmd := NewLonghornCmd(cli)
+	longhornCmd.AddCommand(NewLonghornPrepareForMigration(cli))
```

Comment:
Have we not the code to upgrade from longhorn? Or are this command the first one to do this?

Reply:
This is, as far as I could see, the first Longhorn related command. As I mentioned in the PR description: "This PR only partially fixes sc-65977, the rest of the code is part of the shell script installation and will be raised shortly."